tests/api/test_addon_uploads.py

```diff
@@ -1533,6 +1533,225 @@ def test_extension_delete_previews(base_url, session_auth):
     assert len(get_addon.json()['previews']) == 0
 
 
+@pytest.mark.serial
+@pytest.mark.create_session('api_user')
+def test_edit_default_locale_with_translations(base_url, session_auth):
+    """Change the 'default_locale' of the addon to another locale for which we
+    already have translations for the mandatory fields - i.e. 'name' and 'summary'"""
+    addon = payloads.edit_addon_details['slug']
+    # list all the addon translations and try to set them as the default locale
+    available_translations = ['de', 'fr', 'ro']
+    for item in available_translations:
```

Comment:
Could you name this to be a bit more representative of what the data is? Maybe locale, or language?

Reply:
I've changed the name from `item` to `locale` in all the tests from this set that used this structure. 

Note: the test failure is unrelated (I need to change a locator in the `user_serial_tests`) and I'll fix the issue in another PR.